support/metrics/metricsTracker.go

```diff
@@ -31,6 +33,9 @@ type MetricsTracker struct {
 	botStartTime        time.Time
 	isDisabled          bool
 	updateEventSentTime *time.Time
+
+	// uninitialized
+	handlers []api.TradeMetricsHandler
```

Comment:
this should take in a single pre-initialized pointer to an instance of `tradeMetricsHandler`